### ISSUE TITLE
Compatibility with Rails 6.1

### ISSUE THREAD

**Issue by cseelus** [salsify/goldiloader#97]
Hello Salsify Team. Goldiloader seems to not be compatible with Rails 6.1, which was released today:

```
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    goldiloader (~> 3.1.1) was resolved to 3.1.1, which depends on
      activerecord (>= 4.2, < 6.1)
```

Is there a specific reason why it depends on AR < 6.1?

**Comment by jturkel** [comment]
Thanks for the heads up @cseelus! I'll get a goldiloader release out this morning.

**Comment by jturkel** [comment]
Rails 6.1 support was released in goldiloader 3.2.0 🎉

**Comment by cseelus** [comment]
Thanks, that was fast! 🥳
